src/commands/BootstrapCommand.js

```diff
@@ -201,6 +201,7 @@ export default class BootstrapCommand extends Command {
           const match = find(this.packages, { name: dependency});
           return match && filteredPackage.hasMatchingDependency(match);
         })
+        .map((dependency) => dependency.replace(/^(@.*\/)(.*)$/, "$2"))
```

Comment:
i didn't even think to use the packageGraph! i'll update what i have now to use that.